docs/publishing.md

```diff
@@ -23,15 +23,15 @@ version bumped.
 
 5. Run `npm run pre-release`.
 
-This will:
-  - copy components from `src/` to `packages/` and run tests
-  - add [vendor prefixes](https://github.com/postcss/autoprefixer) to CSS in `packages/`
-  - build "all" Sass and JavaScript files into `dist/`
-  - copy components from `src/` to `dist/components/` and run tests
+  This will:
```

Comment:
this is broken when rendered

Reply:
How did you test it? I'm using Markdown Preview in Atom and it looks okay in that.

Reply:
Have a look now @igloosi